app/src/main/java/com/infomaniak/mail/ui/main/menu/MoveFragment.kt

```diff
@@ -104,4 +124,33 @@ class MoveFragment : MenuFoldersFragment() {
             },
         )
     }
+
+    private fun setSearchBarUi(allFolders: List<Folder>, currentFolderId: String) = with(binding) {
+        searchResultsAdpater = FolderAdapter(
+            isInMenuDrawer,
+            currentFolderId,
+            shouldIndent = false,
+            onFolderClicked = ::onFolderSelected,
+        )
+        searchResultsList.adapter = searchResultsAdpater
+
+        searchInputLayout.trackSearchEndIconClick()
```

Comment:
I don't think we want to use the same matomo as the one from the search, these two parts are pretty different, shouldn't we make a new one instead?

Reply:
Well I have spent a bit of time thinking of this, and the reasonning was : 
We track the user intention to know if a certain button or feature is used. Here in both case the feature is deleting the search text, or validating by clicking on the keyboard. The intention behind doesn't change from one context to the other

Reply:
However we could add one specific Matomo to know if the feature of the move filter itself is used